recipes/libcamera/meta.yaml

```diff
@@ -0,0 +1,111 @@
+{% set libcamera_version = "0.4.0" %}
+{% set libcamera_rpi_version = "0.4.0+rpt20250213" %}
+{% set rpi_url_safe_version = libcamera_rpi_version | urlencode %}
+
+package:
+  name: libcamera
+  version: {{ libcamera_version }}
+
+source:
+{% if variant == "upstream" %}
+  git_url: https://git.libcamera.org/libcamera/libcamera.git
+  git_rev: v{{ libcamera_version }}
+{% elif variant == "rpi_fork" %}
+  url: https://github.com/raspberrypi/libcamera/releases/download/v{{ rpi_url_safe_version }}/libcamera-{{ rpi_url_safe_version }}.tar.xz
```

Comment:
```suggestion
  # In conversations with upstream
  # INSERT LINK TO CONVERSATION
  # We found that they attempt to maintain ABI compatibility
  # Thus we build Raspberry-Pi specific modifications as as a variant instead of a separate package
  url: https://github.com/raspberrypi/libcamera/releases/download/v{{ rpi_url_safe_version }}/libcamera-{{ rpi_url_safe_version }}.tar.xz
```